dll/win32/comctl32/edit.c

```diff
@@ -5181,7 +5179,8 @@ static LRESULT CALLBACK EDIT_WindowProc(HWND hwnd, UINT msg, WPARAM wParam, LPAR
     /* IME messages to make the edit control IME aware */
     case WM_IME_SETCONTEXT:
 #ifdef __REACTOS__
-        if (FALSE) /* FIXME: Condition */
+        /* Korean doesn't want composition window */
+        if (PRIMARYLANGID(LOWORD(GetKeyboardLayout(0))) == LANG_KOREAN)
```

Comment:
Please capture the return value of the `GetKeyboardLayout` before using it inside these macros (+ same comment for the two other places where this is done).